datahub-web-react/src/app/entity/shared/tabs/Dataset/AccessManagement/AccessManagement.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, Divider, Table } from 'antd';
+import { useBaseEntity } from '../../../EntityContext';
+import { GetDatasetQuery, useGetExternalRolesQuery } from '../../../../../../graphql/dataset.generated';
+import { useGetMeQuery } from '../../../../../../graphql/me.generated';
+import { handleAccesslRoles } from './utils';
+import AccessManagerDescription from './AccessManagerDescription';
+
+const StyledTable = styled(Table)`
+    overflow: inherit;
+    height: inherit;
+
+    &&& .ant-table-cell {
+        background-color: #fff;
+    }
+    &&& .ant-table-thead .ant-table-cell {
+        font-weight: 600;
+        font-size: 12px;
+        color: '#898989';
+    }
+    &&
+        .ant-table-thead
+        > tr
+        > th:not(:last-child):not(.ant-table-selection-column):not(.ant-table-row-expand-icon-cell):not([colspan])::before {
+        border: 1px solid #f0f0f0;
+    }
+` as typeof Table;
+const Accessprovision = styled(Divider)`
```

Comment:
let's just call this StyledDivider

Reply:
called styledsection in place of divider could you please suggest if any other tag